web/frontend/src/api/slam.ts

```diff
@@ -69,3 +69,14 @@ export const getSLAMPosition = (client: Client, name: string) => {
     ));
   });
 };
+
+export const getLatestMapInfo = (client: Client, name: string) => {
+  const request = new slamApi.GetLatestMapInfoRequest();
+  request.setName(name);
+
+  return new Promise<commonApi.Pose | undefined>((resolve, reject) => {
```

Comment:
[major] getLatestMapInfo does not use Pose

Reply:
Working on this now (missed it earlier). Problem is that in order to call client.slamService.getLatestMapInfo, the function needs to be included in the auto-generated file rdk/web/frontend/node_modules/@viamrobotics/sdk/dist/gen/service/slam/v1/slam_pb_service.d.ts, and I have no idea how to get it there... similar issue, btw, with rdk/web/frontend/node_modules/@viamrobotics/sdk/dist/gen/service/slam/v1/slam_pb.d.ts